api/chat.go

```diff
@@ -0,0 +1,59 @@
+package api
+
+import (
+	"log"
+
+	"github.com/gin-gonic/gin"
+	"github.com/goccy/go-json"
+	"github.com/mhghw/fara-message/db"
+)
+
+func NewDirectChatRequest(c *gin.Context) {
+	var requestBody NewDirectChat
```

Comment:
function name should a NewDirectChat
strcut name should be NewDirectChatRequest